### PR TITLE
Write a REVISION file in the ios directory to aid in version checks

### PR DESCRIPTION
**Pull request by chinmaygarde**

### CODE REVIEW

File: packages/flutter_tools/lib/src/commands/ios.dart [hunk 1]
```diff
@@ -131,7 +131,12 @@ class IOSCommand extends FlutterCommand {
     // Step 3: Populate the Local.xcconfig with project specific paths
     _setupXcodeProjXcconfig(path.join(xcodeprojPath, "Local.xcconfig"));
 
-    // Step 4: Tell the user the location of the generated project.
+    // Step 4: Write the REVISION file
+    File revisionFile = new File(path.join(xcodeprojPath, "REVISION"));
+    revisionFile.createSync(recursive: false);
```
Comment (devoncarew) [inline packages/flutter_tools/lib/src/commands/ios.dart]: I think the default is false, so we don't need to specify one.

**Comment by devoncarew** [comment]
We're copying the value from the engine version from packages/sky_engine. Will this be up-to-date with the engine version in the xcode zip? If so, lgtm.

**Comment by chinmaygarde** [comment]
> We're copying the value from the engine version from packages/sky_engine. Will this be up-to-date with the engine version in the xcode zip?

Yeah, those versions should be consistent. Rebased the `recursive` named argument removal.

**Comment by devoncarew** [comment]
lgtm

**Pull request merged**